charts/internal/e2e-resources/templates/job-fails.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.failingJob.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-failjob
+spec:
+  activeDeadlineSeconds: 200
+  completions: 4
+  parallelism: 2
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: main
+        image: docker.io/library/bash:5
+        command: ["bash"]
+        args:
+        - -c
+        - echo "Hello world! I'm going to exit with 42 to simulate a software bug." && sleep 5 && exit 42
+  backoffLimit: 6
```

Comment:
`backoffLimit` is capping retries to [6 minutes](https://kubernetes.io/docs/concepts/workloads/controllers/job/#pod-backoff-failure-policy), but `activeDeadlineSeconds` [takes precedence](activeDeadlineSeconds) so job will fail after 200 seconds with `reason: DeadlineExceeded`

Consider removing the unused `backoffLimit`

Reply:
I think `backoffLimit` might be # of tries, not # of minutes. So I think we're good.